cmd/flowrunner/testdata/flows/all_actions_test.json

```diff
@@ -670,7 +670,6 @@
                             "gender": {
                                 "category": "Male",
                                 "created_on": "2000-01-01T00:00:00.000000000-00:00",
-                                "input": "",
```

Comment:
result.input is now nullable if there is no input - but will be omitted when null - included when ""